svm/src/account_overrides.rs

```diff
@@ -4,6 +4,8 @@ use {
 };
 
 /// Encapsulates overridden accounts, typically used for transaction simulations
+/// Account overrides are not used when loading the fee payer account, durable
```

Comment:
If there a technical reason for that ? Or are we just waiting for it to be implemented ? 
Seems if we are just waiting for it to be the comment might easily be outdated and misleading. 
I can rebase and update my PR once this is merged as to remove the comment.

See : 
https://github.com/anza-xyz/agave/issues/2210
https://github.com/anza-xyz/agave/pull/2375

Reply:
Just waiting for it to be implemented, I suppose (with the exception of the instruction sysvar). I actually didn't realize that before the refactoring fee payer overrides were already supported. We can merge yours in first and I'll rebase and update the comment

Reply:
> Just waiting for it to be implemented, I suppose (with the exception of the instruction sysvar). I actually didn't realize that before the refactoring fee payer overrides were already supported. We can merge yours in first and I'll rebase and update the comment

That would be amazing ! Super happy to have first contribution to Agave <3

Reply:
> Btw how did you find this comment so fast?

haha - just lucky timing - clicked on the pull requests and looked at yours because it was related to account loading :D